docs/api/desktop-capturer.md

```diff
@@ -119,3 +119,9 @@ captured.
 Returns `Promise<DesktopCapturerSource[]>` - Resolves with an array of [`DesktopCapturerSource`](structures/desktop-capturer-source.md) objects, each `DesktopCapturerSource` represents a screen or an individual window that can be captured.
 
 [`navigator.mediaDevices.getUserMedia`]: https://developer.mozilla.org/en/docs/Web/API/MediaDevices/getUserMedia
+
+### Caveats
+
+`navigator.mediaDevices.getUserMedia` does not work on macOS for audio capture due to a fundamental limitation whereby apps that want to access the system's audio require a signed kernel. Chromium, and by  extension Electron, does not provide this.
```

Comment:
What's a signed kernel?

Can we link to macOS docs that explain this?

Reply:
done!